app/services/platforms/twitch.ts

```diff
@@ -313,11 +346,24 @@ export class TwitchService
     this.twitchTagsService.actions.setTags(tags);
     const hasPermission = await this.hasScope('channel:manage:broadcast');
     const scopedTags = hasPermission ? tags : undefined;
+
+    // Twitch seems to require you to add a label with disabled to remove it
+    const labels = this.twitchContentClassificationService.options.map(option => ({
+      id: option.value,
+      is_enabled: contentClassificationLabels.includes(option.value),
+    }));
+
     await Promise.all([
```

Comment:
I'm not sure why we have `Promise.all` here, lol. Should be just `await this.requestTwitch`

Reply:
i think there previously may have been other steps or something 🤔